lib/posexional/file.ex

```diff
@@ -76,6 +76,34 @@ defmodule Posexional.File do
     end)
   end
 
+  @spec stream(Enumerable.t(), %Posexional.File{}) :: Enumerable.t()
+  def stream(str, _file = %{separator: separator, rows: rows}) do
+    str
+    |> Stream.concat([separator])
+    |> Stream.transform("", &stream_split(&1, &2, separator))
+    |> Stream.reject(&match?("", &1))
```

Comment:
Non sarebbe più corretto usare semplicemente `&(&1 === "")`? O match fa cose particolari?

Reply:
basta `==` per il confronto con una stringa..

Reply:
fidarsi è bene, non fidarsi è meglio. E poi credo che === sia più performante, perché esclude il type casting. Super flex comunque...

Reply:
Vale solo per i numeri il casting, questa e' la [doc](https://www.erlang.org/doc/reference_manual/expressions.html) di erlang:
```
When comparing an integer to a float, the term with the lesser precision is converted into the type of the other term, unless the operator is one of =:= or =/=.
```
Per il resto penso proprio che siano la stessa cosa, e in elixir credo sia piu' idiomatico `==`.